.github/workflows/integration-setup.yaml

```diff
@@ -17,31 +17,48 @@ jobs:
       run: |
         export DEBIAN_FRONTEND=noninteractive
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends curl helm
+        sudo apt-get install -y --no-install-recommends curl unzip
+    - name: setup helm
+      uses: azure/setup-helm@v3
     - name: cleanup
       run: |
-        helm list release -n ${{ secrets.DEV_SANDBOX_NAMESPACE }} | grep integration | xargs -I env -n helm delete env -n ${{ secrets.DEV_SANDBOX_NAMESPACE }} --debug --timeout 10m0s  
+        helm list -n ${{ secrets.DEV_SANDBOX_NAMESPACE }} | grep integration | cut -f 1 -d " " | xargs -I env -n 1 helm delete env -n ${{ secrets.DEV_SANDBOX_NAMESPACE }} --debug --timeout 10m0s  
```

Comment:
Should we maybe have a different worflow to cleanup the database vs cleanup old failed tests, in case we want new tests to proceed but still debug failed ones?